#goal definition
int32 order
---
#result definition
bool result
---
#feedback
float64 x 
float64 y
float64 z
int32 brick_id 